spec/factories/users.rb

```diff
@@ -270,16 +269,14 @@
       confirmed_at { Time.zone.now.round }
 
       after :build do |user|
-        profile = create(
+        create(
```

Comment:
it would be nice if we could do this as a `build()` instead of a `create()` but I'm guessing we rely on DB associations and scopes for these, so easier to just always create

Reply:
Could you say more about the reason for preferring `build()` here?  Is it because you’d expect `build()` to be faster than `create()`?  Or is it about wanting to avoid calling `create()` inside of an after build hook?

Reply:
1. yes, `build()` should be faster than `create()` because `build` is in-memory only
2. the other reason is that `after_build` means that if an upstream caller of this constructor only builds, intending to be in-memory only, now by calling `create` this will hit the DB, which may be unexpected and unintended

Reply:
I tried making the suggested change, but it broke a spec in the delete account controller spec.  I am going to merge the PR as is.